Assets/Scripts/PlayerMovement.cs

```diff
@@ -15,6 +15,7 @@ public class PlayerMovement : MonoBehaviour
     Vector2 moveInput;
     Rigidbody2D myRigidbody;
     CapsuleCollider2D myCapsuleCollider;
+    GameObject checkpoint;
```

Comment:
If we are only going to save the position, then we should use Vector3.
```suggestion
    Vector3 position;
```